### PR TITLE
Updated Manual Scripts

### PR DESCRIPTION
**Pull request by kowsisoundhar12**
updated Manual SCripts:
    'Test_TC_DD_1_10',
    'Test_TC_DD_2_2',
    'Test_TC_DD_3_1',
    'Test_TC_DD_3_2',
    'Test_TC_DD_3_5',
    'Test_TC_DD_3_6',
    'Test_TC_DD_3_9',
    'Test_TC_DD_3_10',
    'Test_TC_DD_3_11',
    'Test_TC_DD_3_12',
    'Test_TC_DD_3_13',
    'Test_TC_BDX_1_2',
    'Test_TC_BDX_2_1',
    'Test_TC_BDX_2_2',
    'Test_TC_BDX_2_3',
    'Test_TC_BR_1',
    'Test_TC_BR_2',
    'Test_TC_BR_3',
    'Test_TC_BRAC_2_1',
    'Test_TC_BRAC_2_2',
    'Test_TC_DM_1_2',
    'Test_TC_DM_1_4',
    'Test_TC_DM_2_3',
    'Test_TC_DM_2_4',
    'Test_TC_DM_3_2',
    'Test_TC_DM_3_4',
    'Test_TC_DIAG_LOG_1_1',
    'Test_TC_DIAG_LOG_1_2',
    'Test_TC_DIAG_LOG_1_3',
    'Test_TC_DESC_1_1',
    'Test_TC_ETHDIAG_1_2',
    'Test_TC_GC_1_2',
    'Test_TC_GC_1_3',
    'Test_TC_GENDIAG_1_1',
    'Test_TC_GENDIAG_1_2',
    'Test_TC_GENDIAG_2_1',
    'Test_TC_I_2_2',
    'Test_TC_ILL_2_2',
    'Test_TC_IDM_1_1',
    'Test_TC_IDM_1_2',
    'Test_TC_IDM_6_1',
    'Test_TC_IDM_6_2',
    'Test_TC_IDM_6_3',
    'Test_TC_IDM_6_4',
    'Test_TC_MC_3_12',
    'Test_TC_MC_3_13',
    'Test_TC_MC_4_1',
    'Test_TC_MC_8_2',
    'Test_TC_MC_9_2',
    'Test_TC_MC_10_2',
    'Test_TC_MC_10_3',
    'Test_TC_MC_10_4',
    'Test_TC_MC_10_5',
    'Test_TC_MC_10_6',
    'Test_TC_MF_1_1',
    'Test_TC_MF_1_2',
    'Test_TC_MF_1_3',
    'Test_TC_MF_1_5',
    'Test_TC_MF_1_6',
    'Test_TC_MF_1_7',
    'Test_TC_MF_1_8',
    'Test_TC_MF_1_9',
    'Test_TC_MF_1_10',
    'Test_TC_MF_1_11',
    'Test_TC_MF_1_12',
    'Test_TC_MF_1_13',
    'Test_TC_MF_1_14',
    'Test_TC_MF_1_15',
    'Test_TC_MF_1_16',
    'Test_TC_MF_1_17',
    'Test_TC_MF_1_18',
    'Test_TC_MF_1_19',
    'Test_TC_MF_1_20',
    'Test_TC_MF_1_21',
    'Test_TC_MF_1_22',
    'Test_TC_MF_1_23',
    'Test_TC_MF_1_24',
    'Test_TC_MF_1_25',
    'Test_TC_MF_1_26',
    'Test_TC_MF_1_27',
    'Test_TC_MF_1_28',
    'Test_TC_MOD_1_2',
    'Test_TC_MOD_2_1',
    'Test_TC_MOD_2_2',
    'Test_TC_MOD_3_1',
    'Test_TC_MOD_3_2',
    'Test_TC_MOD_3_3',
    'Test_TC_SU_1_1',
    'Test_TC_SU_2_1',
    'Test_TC_SU_2_2',
    'Test_TC_SU_2_3',
    'Test_TC_SU_2_4',
    'Test_TC_SU_2_5',
    'Test_TC_SU_2_6',
    'Test_TC_SU_2_7',
    'Test_TC_SU_3_1',
    'Test_TC_SU_3_2',
    'Test_TC_SU_3_3',
    'Test_TC_SU_3_4',
    'Test_TC_SU_4_1',
    'Test_TC_SU_4_2',
    'Test_TC_PSCFG_1_1',
    'Test_TC_PSCFG_2_1',
    'Test_TC_SWDIAG_1_2',
    'Test_TC_WIFIDIAG_1_2',
    'Test_TC_WIFIDIAG_2_1',
    'Test_TC_WNCV_2_3',
    'Test_TC_WNCV_6_1',
    'Test_TC_FLW_3_1',
    'Test_TC_OCC_2_3',
    'Test_TC_OCC_2_4',
    'Test_TC_OCC_3_1',
    'Test_TC_OO_2_4',
    'Test_TC_PRS_2_2',
    'Test_TC_PRS_2_3',
    'Test_TC_PS_2_2',

### CODE REVIEW

**Comment by kowsisoundhar12** [comment]
@bzbarsky-apple Your PR is giving conflict to following Manual scripts [Test_TC_IDM_1_2 &Test_TC_PSCFG_1_1] ,have you changed this Manual script intentionally in ur PR ?

Note : please refer your PR link here 
https://github.com/project-chip/connectedhomeip/pull/16456/

**Comment by bzbarsky-apple** [comment]
> have you changed this Manual script intentionally in ur PR ?

"Intentionally" in the sense that it was updated to use the new spec names for the relevant commands, along with everything else in the tree.

Looking at this PR, you probably want to rebase to master, use this PR's versions of `Test_TC_IDM_1_2.yaml` and `Test_TC_PSCFG_1_1.yaml` but then replace `ClientGeneratedCommandList` with `AcceptedCommandList` and `ServerGeneratedCommandList` with `GeneratedCommandList` in both.

**Comment by kowsisoundhar12** [comment]
@bzbarsky-apple thanks for letting me know ,sure i will rebase the PR with the  changes u mentioned

File: src/app/tests/suites/certification/Test_TC_IDM_1_2.yaml [hunk 1]
```diff
@@ -23,224 +23,251 @@ tests:
...
+          [1646261193889] [5094:213098] CHIP: [DMG]
+          [1646261193889] [5094:213098] CHIP: [DMG]                 },
+          [1646261193889] [5094:213098] CHIP: [DMG]
+          [1646261193889] [5094:213098] CHIP: [DMG]         ],
+          [1646261193889] [5094:213098] CHIP: [DMG]
+          [1646261193889] [5094:213098] CHIP: [DMG]         SuppressResponse = true,
+          [1646261193889] [5094:213098] CHIP: [DMG]         InteractionModelRevision = 1
+          [1646261193889] [5094:213098] CHIP: [DMG] }
+          [1646261193889] [5094:213098] CHIP: [TOO] Endpoint: 0 Cluster: 0x0000_0028 Attribute 0x0000_FFF9DataVersion: 759594214
+          [1646261193889] [5094:213098] CHIP: [TOO]   ClientGeneratedCommandList: 0 entries
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_IDM_1_2.yaml:272]: This is not the right output.

Comment (kowsisoundhar12) [reply src/app/tests/suites/certification/Test_TC_IDM_1_2.yaml]: MOdified the testscript

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
Fast-tracking test addition.

**Pull request merged**